docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -11,19 +10,23 @@
   },
   {
    "cell_type": "markdown",
-   "id": "4aecd155",
    "metadata": {},
    "source": [
     "## Introduction\n",
     "### Data Representation\n",
-    "Web Dataset is a dataset representation that heavily optimizes networked accessed storage performance. At its simplest, it stores the whole dataset in one tarball file, where individual samples are kept under the files with the same names but different extensions. This approach improves drive access caching on the RAM, since the data is represented sequentially.\n",
+    "Web Dataset is a dataset representation that heavily optimizes networked accessed storage performance. At its simplest, it stores the whole dataset in one tarball file, where individual samples are kept under the files with the same names but different extensions. This approach improves drive access caching on the RAM, since the data is represented sequentially."
```

Comment:
Note being very well versed in WebDataset, I find this a bit unclear. Does it mean this?
```suggestion
    "Web Dataset is a dataset representation that heavily optimizes networked accessed storage performance. At its simplest, it stores the whole dataset in one tarball file, where each sample is represented by one or more entries with the same same name but different extensions. This approach improves drive access caching in RAM, since the data is represented sequentially."
```